src/Renderer/HorizonRenderer.cpp

```diff
@@ -11,6 +11,86 @@
 
 #include <algorithm>
 
+#include <iostream>
```

Comment:
Do we need this include? I haven't seen a function that uses it, but haven't tried to remove it yet.